samples/sensor/fxos8700/README.rst

```diff
@@ -6,20 +6,70 @@ FXOS8700 Accelerometer/Magnetometer Sensor
 Overview
 ********
 
-A sensor application that demonstrates how to use the fxos8700 data ready
-interrupt to read accelerometer/magnetometer data synchronously.
+This sample application shows how to use the FXOS8700 driver.
+The driver supports FXOS8700 accelerometer/magnetometer and
+MMA8451Q, MMA8652FC, MMA8653FC accelerometers.
 
 Building and Running
 ********************
 
-This project outputs sensor data to the console. It requires an fxos8700
-sensor, which is present on the :ref:`frdm_k64f`, :ref:`frdm_kw41z`, and
-:ref:`hexiwear_k64` boards.  It does not work on QEMU.
+This project outputs sensor data to the console. FXOS8700
+sensor is present on the :ref:`frdm_k64f`, :ref:`frdm_kw41z`, and
+:ref:`hexiwear_k64` boards. Accelerometer only devices are
+present on the :ref:`frdm_kl25z`, :ref:`bbc_microbit`, and
+:ref:`reel_board` boards. It does not work on QEMU.
+
+For accelerometer devices like MMA8451Q, MMA8652FC or MMA8653FC,
+open the project configuration file :file:`prj.conf` and set the following
+variables:
```

Comment:
Let's provide a new `prj_accel.conf` file instead of asking users to manually edit the existing one.

Reply:
When you've finished this sample, I'll create something fit the micro: bit. 
I'll use the display to give some graphical output.

Reply:
done